sdks/python/apache_beam/yaml/generate_yaml_docs.py

```diff
@@ -86,10 +86,16 @@ def _fake_value(name, beam_type):
     raise ValueError(f"Unrecognized type_info: {type_info!r}")
 
 
+EXCLUDE_ARGS = ['args', 'kwargs']
```

Comment:
We can't always filter these out, e.g. https://beam.apache.org/releases/yamldoc/current/#pytransform

Reply:
Ah.. right.. let me rethink